src/api/app/assets/javascripts/webui2/monitor.js

```diff
@@ -91,3 +91,57 @@ function jobPlot(data) {
       legend: { noColumns: 3, position: "ne", container: "#legend-jobs" }
     });
 }
+
+function processProgressBar(id, item)
+{
+  var delta = item["delta"];
+
+  var container = $('#p' + id);
+  var el_text = container.find(".monitorpb_text");
+  var ctrl = container.find(".progress-bar");
+
+  var logfileinfo = $("#worker-display option:selected").val();
+
+  if (delta) {
+    el_text.text(item[logfileinfo]);
+    ctrl.css("width", delta + "%").attr("aria-valuenow", delta);
+    var url = $('#workers').data('buildLogPath');
+    url = url.replace('ARCH', item["arch"]);
+    url = url.replace('PROJECT', item["project"]);
+    url = url.replace('PACKAGE', item["package"]);
+    url = url.replace('REPOSITORY', item["repository"]);
+
+    el_text.attr("href", url);
+
+    if (delta <= 50) {
```

Comment:
It's a bit difficult to read. I would suggest changing the font color based on the delta.

![difficult-to-read](https://user-images.githubusercontent.com/1102934/58871568-90a4af00-86c2-11e9-8876-6cd579d6b834.png)

It's not a blocker, but it would be nice.

Reply:
can you please provide me some example what you mean?

Reply:
I'm not so sure about which color we should use here. As we discussed in the chat, let's leave it as it is for now.